client/src/components/ProfilePage/profile.js

```diff
@@ -40,17 +38,38 @@ const Profile = (props) => {
         </div>
       </div>
       <img src="https://cdn-icons-png.flaticon.com/128/2102/2102633.png"
-            alt="Profile"
-            className="rounded-circle img-fluid profile-picture mb-3 mb-md-0"
-          />
+        alt="Profile"
+        className="rounded-circle img-fluid profile-picture mb-3 mb-md-0"
+      />
       <div className="row">
         <pre className="col-12 text-light bg-dark p-4">
           {JSON.stringify(user, null, 2)}
         </pre>
       </div>
+      <div>
+        <button onClick={handleFavoriteClick}>Your Favorited Movie List</button>
+
 
-      
+
+      </div>
+      {favMovie && favMovie.length > 0 && ( // Check if favMovie exists and has items
+        <div>
+         
+         <ul>
+        <div className="movie-list">
```

Comment:
hmm I don't think its valid HTML to have a `<div>` as a child of `<ul>` - does this render correctly?

Reply:
It does not throw me the error. What might be the better practice?